R/aes-evaluation.r

```diff
@@ -55,6 +121,44 @@
 #' # Use stage to modify the scaled fill
 #' ggplot(mpg, aes(class, hwy)) +
 #'   geom_boxplot(aes(fill = stage(class, after_scale = alpha(fill, 0.4))))
+#'
+#' # Making a proportional stacked density plot
+#' ggplot(mpg, aes(cty)) +
+#'   geom_density(
+#'     aes(colour = factor(cyl),
```

Comment:
Needs to follow https://style.tidyverse.org/syntax.html#long-lines. You might want to use https://github.com/lionel-/codegrip to make the transformations easier. (Although it probably doesn't work inside roxygen2 comments yet)